akka-bbb-apps/src/main/scala/org/bigbluebutton/core/apps/breakout/ChangeUserBreakoutReqMsgHdlr.scala

```diff
@@ -58,7 +58,6 @@ trait ChangeUserBreakoutReqMsgHdlr extends RightsManagementTrait {
 
         //Send notification to moved User
         for {
-          roomFrom <- breakoutModel.rooms.get(msg.body.fromBreakoutId)
           roomTo <- breakoutModel.rooms.get(msg.body.toBreakoutId)
```

Comment:
```suggestion
	  roomFrom <- breakoutModel.rooms.get(msg.body.fromBreakoutId)
          roomTo <- breakoutModel.rooms.get(msg.body.toBreakoutId)
```

I suggest to keep the `roomFrom` as a validation step.